### DISCUSSION TITLE
Problem During pulling the image from docker hub

### DISCUSSION THREAD

**Discussion by Riturajvats** [apache/airflow#24887] [Q&A]
Image got build successfully  after that I pushed that image with the name repostiory_name/arflow3:2.3.2_v1 to docker hub but when I trying to pull the image from docker hub it throwing error that is 

Error:- "failed to register layer: max depth exceeded"

And I'm getting this error when I'm trying to pull this particular image.
![Screenshot (137)](https://user-images.githubusercontent.com/54302152/177665745-a2ff68a5-e843-4658-ad10-4727f193c802.png)

**Comment by potiuk** [comment]
You have your image build wrongly https://stackoverflow.com/questions/47272611/docker-max-depth-exceeded - you will find some explanations there.

**Reply by Riturajvats** [reply]
![image_2022-07-07_212055476](https://user-images.githubusercontent.com/54302152/177816950-a168b8ea-8792-426a-a39e-f1184f2e6feb.png)
But here in the output it's showing the image got build completely .

**Reply by potiuk** [reply]
I have no idea - just found the thread by googling. Read it carefully and you will lkely find the solution. Seems that it has something to do with "/" in the name of your image but I have not read it in detail. But you should.
